languages.yaml

```diff
@@ -2501,6 +2501,27 @@ Xml:
       - white
       - green
     chip: '#0060AC'
+XSL:
+  type: markup
+  ascii: |
+    {0}
+    {0}                     :::
+    {0}          ::::       :::  ::::
+    {0}        ::::::      :::   ::::::
+    {0}       :::::        :::     :::::
+    {0}     ::::::        :::       ::::::
+    {0}   ::::::          :::         ::::::
+    {0}   ::::::         :::          ::::::
+    {0}     ::::::       :::        ::::::
+    {0}       :::::     :::        :::::
+    {0}        ::::::   :::      ::::::
+    {0}          ::::  :::       ::::
+    {0}                :::
+    {0}
+  colors:
+    ansi:
+      - cyan
+    chip: '#58A7D7'
```

Comment:
Should be [#EB8CEB](https://github.com/github/linguist/blob/4ea1b93800c27ecb2221d4a9b19d8881e9197544/lib/linguist/languages.yml#L7378)

Reply:
I see, use the [linguist link](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) and find the language. Thanks for getting the numbers!

I new it wasn't a random number, thanks again.

Reply:
done